editor/editor_node.cpp

```diff
@@ -2983,6 +2983,19 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 			}
 
 			if (!p_confirmed) {
+				if (project_run_bar->is_playing()) {
+					if (p_option == RELOAD_CURRENT_PROJECT) {
+						confirmation->set_text(TTR("Terminate project before reloading?"));
+						confirmation->set_ok_button_text(TTR("Terminate & Reload"));
+					} else {
+						confirmation->set_text(TTR("Terminate project before closing?"));
+						confirmation->set_ok_button_text(TTR("Terminate & Quit"));
```

Comment:
We use the "stop" term elsewhere, so I suggest also using it here for consistency:

```suggestion
						confirmation->set_text(TTR("Stop project before reloading?"));
						confirmation->set_ok_button_text(TTR("Stop & Reload"));
					} else {
						confirmation->set_text(TTR("Stop project before closing?"));
						confirmation->set_ok_button_text(TTR("Stop & Quit"));
```